src/PrestaShopBundle/Resources/config/routing/admin/configure/advanced_parameters/experimental_feature.yml

```diff
@@ -0,0 +1,15 @@
+admin_experimental_feature_index:
+    path: /
+    methods: GET
+    defaults:
+        _controller: 'PrestaShopBundle:Admin\Configure\AdvancedParameters\ExperimentalFeature:index'
+        _legacy_controller: AdminExperimentalFeature
+        _legacy_link: AdminExperimentalFeature
```

Comment:
wouldn't it be useful to expose this route, so that it could be used in js if we need to redirect toward this page for some reason ?
(or do we expose them on a per-need basis?)

Reply:
So far it was per-need basis 🤷‍♂️